components/PyTorch/pytorch-kfp-components/README.md

```diff
@@ -0,0 +1,23 @@
+# PyTorch Kubeflow Pipeline Components
```

Comment:
I suggest adding an OWNERS file for `components/PyTorch` folder (or pytorch-kfp-components folder, you can decide).

Example: https://github.com/kubeflow/pipelines/blob/master/OWNERS
Documentation: https://www.kubeflow.org/docs/about/contributing/#owners-files-and-pr-workflow

OWNERS file make any PRs sent against this folder to assign to you for review, instead of us. So it's good for long term maintenance.

Reply:
Added OWNERS file